OurPlanscape/Planscape
# Use map sourceData event to clear and restore selected stands

Removing hacky setTimeout and using map events instead. Cleaning up duplicated stream on treatment-map

To test:

- Go to a treatment plan, select a project area
- Select some stands (either manually or clicking the assignments on the left side expander component)
- Change base layer
- Assign treatments

Verify that the map is still working fine, treatment is assigned properly, etc.

Additionally, navigate back and forward between treatment overview and project area, validate that the layers are loading and showing properly.
